### PR TITLE
fix: update all examples to newest version of bacalhau

### PR DESCRIPTION
**Pull request by aronchick**
Needs to wait for bacalhau v0.28

### CODE REVIEW

File: data-ingestion/index.ipynb [hunk 1]
```diff
@@ -117,7 +117,10 @@
             "outputs": [],
             "source": [
                 "%%bash --out job_id\n",
-                "bacalhau docker run --wait --input-urls http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz:/inputs/train-images-idx3-ubyte.gz ubuntu -- cp -rv /inputs/. /outputs/"
+                "bacalhau docker run \\\n",
+                "    --id-only\n",
```
Comment (lukemarsden) [inline data-ingestion/index.ipynb:121]: this is missing a \ on the end of the line?

**Review by lukemarsden — CHANGES_REQUESTED** [review]
why are these files here now?

molecular-dynamics/openmm/run_openmm_simulation.py
data-engineering/oceanography-conversion/main.py

also one substantive bug

**Comment by lukemarsden** [comment]
thank you for doing this 🙌

**Comment by lukemarsden** [comment]
also please can you check why CI is still red?

File: data-ingestion/index.ipynb (hunk 1 above)
Comment (aronchick) [reply data-ingestion/index.ipynb:121]: yep

**Pull request merged**